lib/grpc-api-assistant/grpc_steps.rb

```diff
@@ -1,3 +1,5 @@
+require 'time'
```

Comment:
[I've been trying to keep all the requires statements in a single place](https://github.com/michaelkeeling/cucumber-grpc-api-assistant/blob/master/lib/grpc-api-assistant/env.rb).

Reply:
Not sure why, but moving the require statement to that file doesn't work. If it's moved to [this](https://github.com/michaelkeeling/cucumber-grpc-api-assistant/blob/master/lib/grpc-api-assistant.rb#L13-L14) file, it does. What do ya think?

Reply:
it looks like the autoloading in either Ruby or Cucumber (or both) changed at some point with the upgrades I just pushed up.  Don't worry about this change.  I'm not 100% what that `.env` file is doing now 😛